src/components/layout-item/usage/Basic.md

```diff
@@ -0,0 +1,9 @@
+```html
```

Comment:
For best practices, what is the distinction between using this and using List with selection mode none?